IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -601,7 +699,12 @@ public static AnyBitmap CreateMultiFrameGif(IEnumerable<AnyBitmap> images)
             {
                 using (var memoryStream = new System.IO.MemoryStream())
                 {
-                    Image.Save(memoryStream, new SixLabors.ImageSharp.Formats.Bmp.BmpEncoder());
+                    // BMP transparency support only 32 bits
```

Comment:
says only support 32 bits per pixel in comments, but implemented as 24

Reply:
This is just the note.
When loading Image with SixLabors.ImageSharp.Image<SixLabors.ImageSharp.PixelFormats.Rgb24> it'll loss the transparency at first.
Maybe the better way is just removed the comment that may confusion.
What do you think?